webhook_to_fedora_messaging/endpoints/parser/parser.py

```diff
@@ -0,0 +1,53 @@
+from  flask import Request, request
+from werkzeug.datastructures.headers import EnvironHeaders
+import hashlib
+import hmac
+from webhook_to_fedora_messaging_messages.github.github import GithubMessageV1
+from ..util import fas_by_github
+from ...exceptions.exceptions import SignatureMatchError, SecretNotFoundError
+
+
+def msg_parser(service_type: str, secret: str):
+    if service_type.lower() == "github":
+        return _github_parser(secret)
+    else:
+        raise ValueError
+
+
+def _github_parser(secret: str) -> GithubMessageV1:
```

Comment:
Please put this Github-specific code into a `github.py` module (also put the `verify_signature()` function there as it is Github-specific as well).